src/resolve.rs

```diff
@@ -380,10 +380,15 @@ fn flatten(chunks: &mut Chunks) {
 }
 
 /// Characters that can be escaped.
-pub fn is_escapable(c: char, verb: bool) -> bool {
+///
+/// In read mode (`read_char = true`), colons are also converted to an unescaped string to keep
+/// compatiblity with Zotero. Zotero escapes colons when exporting verbatim fields. This crate
+/// doesn't escape colons when exporting.
+pub fn is_escapable(c: char, verb: bool, read_char: bool) -> bool {
     match c {
-        '{' | '}' | '\\' | ':' => true,
+        '{' | '}' | '\\' => true,
         '&' | '%' | '$' | '_' if !verb => true,
+        ':' if read_char => true,
```

Comment:
The colon would not be escaped when writing with this condition, even if the field is not to be dumped in verbatim mode. Instead, I suggest the following:

```suggestion
        ':' if read_char || !verb => true,
```

Reply:
Thanks for your suggestion! I added this.

One question: Is there a specific reason for escaping all colons (in non-verbatim mode)? So far, I thought colons are not a problem in bibtex/biblatex files. 

However, I found the following issue: https://github.com/JabRef/jabref/issues/5982 If I understand it correctly, colons are only a problem in the bibtex key.

Thanks for your clarification and your help.

Reply:
Oops I overlooked your comment. You are right, as far as I see, the colon need not be escaped in values but only in keys. This means that the code would have been right after all, without my suggestion. As we've seen, there is no harm in being permissive about what escapes to accept when reading. If this is no urgent matter I'll revert back to your original PR in the next bugfix release.

Reply:
Thanks for your feedback! No, this isn't a urgent matter. 😄